tests/test_schema.py

```diff
@@ -2174,6 +2174,112 @@ def test_schema_get_migration_43(self):
 
         self._assert_migration_consistency(schema)
 
+    def test_schema_get_migration_44(self):
+        schema = r'''
+        type Foo {
+            property val -> str;
+            property comp := count((
+                # Use an alias in WITH block in a computable
+                WITH x := .val
+                # Use an alias in SELECT in a computable
+                SELECT y := Bar FILTER x = y.val
+            ))
+        }
+
+        type Bar {
+            property val -> str;
+        }
+        '''
+
+        self._assert_migration_consistency(schema)
+
+    def test_schema_get_migration_45(self):
+        # The error should be about usage of mutations in computable,
+        # not about aliases.
+        with self.assertRaisesRegex(
+            errors.SchemaDefinitionError,
+            "invalid mutation in computable property 'comp'"
```

Comment:
Observation: the phrasing of this error implies that there is a "valid mutation" in a computable property, when, in fact, all mutations are banned.

Reply:
Reworded.